### PR TITLE
Updated docs on setting the CTR URI property in dataflow 

### PR DESCRIPTION
**Pull request by cppwfs**
Resolves https://github.com/spring-cloud/spring-cloud-dataflow/issues/3781

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -25,7 +25,7 @@
     invariant "^2.2.4"
     semver "^5.5.0"
 
-"@babel/core@^7.2.2", "@babel/core@^7.8.4", "@babel/core@^7.8.7":
+"@babel/core@7.9.0", "@babel/core@^7.2.2", "@babel/core@^7.5.5", "@babel/core@^7.9.0":
```
Comment (ilayaperumalg) [inline yarn.lock:28]: Not sure if we want to check-in this file.

File: content/documentation/pages/5-feature-guides/3-batch/6-composed-task.md [hunk 2]
```diff
@@ -114,24 +114,10 @@ Repeat this registration for `task-a`, `task-b`, `task-c`, `task-d`, `task-e`, a
...
-          composed-task-runner
-         ```
-   1. For the `Type` field select: `Task`
-   1. Enter the following into the `URI` field: 1. If using Maven:
-      `maven://org.springframework.cloud.task.app:composedtaskrunner-task:2.1.3.RELEASE` 1. if using Docker:
-      `docker:springcloudtask/composedtaskrunner-task:2.1.3.RELEASE`
-      The result should look something like:
-      ![Composed Task Graph](images/SCDF-composed-task-register.png)
-1. Press the `Register the Applications(s)` button.
+[[note]]
```
Comment (ilayaperumalg) [inline content/documentation/pages/5-feature-guides/3-batch/6-composed-task.md:117]: I think we still need to add the default registration flow, no?

File: yarn.lock (hunk 1 above)
Comment (cppwfs) [reply yarn.lock:28]: DOH!   Thanks for catching this!

File: content/documentation/pages/5-feature-guides/3-batch/6-composed-task.md (hunk 2 above)
Comment (cppwfs) [reply content/documentation/pages/5-feature-guides/3-batch/6-composed-task.md:117]: For 2.6 forward a user will not need to register the CTR anymore.   This registration is handled by https://github.com/spring-cloud/spring-cloud-dataflow/blob/master/spring-cloud-dataflow-server-core/src/main/resources/META-INF/dataflow-server-defaults.yml#L71 and https://github.com/spring-cloud/spring-cloud-dataflow/blob/master/src/templates/kubernetes/server/server-deployment.yaml#L57-L58

Comment (ilayaperumalg) [reply content/documentation/pages/5-feature-guides/3-batch/6-composed-task.md:117]: Sorry, this was a misunderstanding on my part even after reviewing the SCDF PR ;) Thanks @cppwfs !

Comment (cppwfs) [reply content/documentation/pages/5-feature-guides/3-batch/6-composed-task.md:117]: No problem..   Thanks for the review!!!!!!!

**Comment by ilayaperumalg** [comment]
LGTM, merging.

**Pull request merged**